presenter-in-iframe/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>FieldTwin in iframe</title>
+    <style>
+      body {
+        background-color: #ffffff;
+      }
+    </style>
+  </head>
+  <body style="height: 100vh; margin: 0; border: 0">
+    <div style="width: 100%; height: 100%; display: flex; overflow: hidden">
+      <div style="width: 30%; height: 100%; display: flex; flex-flow: column">
+        <div style="width: 100%; flex: 0 1 auto">
```

Comment:
I know it's only a demo but it would be a lot neater to move these into the style tag at L9 and define a few classes. Especially for the L17 style that repeats in 5 other places.